### ISSUE TITLE
Server Crashes NullPointerException BossSpawning

### ISSUE THREAD

**Issue by BugTest85** [Shadows-of-Fire/Apotheosis#666]
Server Crashes when spezifaced Boss Monsters Spawns - three times in a day
ATM 7 ModPack + some useded Mods
```
---- Minecraft Crash Report ----
// Don't do that.

Time: 9/29/22, 3:12 AM
Description: Exception ticking world

java.lang.NullPointerException: Cannot invoke "net.minecraft.network.chat.TextColor.m_131265_()" because the return value of "net.minecraft.network.chat.Style.m_131135_()" is null
	at shadows.apotheosis.adventure.boss.BossEvents.lambda$naturalBosses$0(BossEvents.java:60) ~[Apotheosis-1.18.2-5.6.1.jar%2355!/:5.6.1] {re:classloading}
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?] {re:mixin}
	at shadows.apotheosis.adventure.boss.BossEvents.naturalBosses(BossEvents.java:56) ~[Apotheosis-1.18.2-5.6.1.jar%2355!/:5.6.1] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_1834_BossEvents_naturalBosses_CheckSpawn.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.event.ForgeEventFactory.canEntitySpawn(ForgeEventFactory.java:196) ~[forge-1.18.2-40.1.80-universal.jar%23366!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraftforge.common.ForgeHooks.canEntitySpawn(ForgeHooks.java:1109) ~[forge-1.18.2-40.1.80-universal.jar%23366!/:?] {re:classloading}
	at net.minecraft.world.level.NaturalSpawner.m_47038_(NaturalSpawner.java:183) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,re:classloading,pl:mixin:APP:botania_xplat.mixins.json:MixinNaturalSpawner,pl:mixin:APP:enhancedcelestials.mixins.json:MixinWorldEntitySpawner,pl:mixin:APP:enhancedcelestials.mixins.json:access.WorldEntitySpawnerAccess,pl:mixin:A}
	at net.minecraft.world.level.NaturalSpawner.m_47045_(NaturalSpawner.java:126) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,re:classloading,pl:mixin:APP:botania_xplat.mixins.json:MixinNaturalSpawner,pl:mixin:APP:enhancedcelestials.mixins.json:MixinWorldEntitySpawner,pl:mixin:APP:enhancedcelestials.mixins.json:access.WorldEntitySpawnerAccess,pl:mixin:A}
	at net.minecraft.world.level.NaturalSpawner.m_47029_(NaturalSpawner.java:116) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,re:classloading,pl:mixin:APP:botania_xplat.mixins.json:MixinNaturalSpawner,pl:mixin:APP:enhancedcelestials.mixins.json:MixinWorldEntitySpawner,pl:mixin:APP:enhancedcelestials.mixins.json:access.WorldEntitySpawnerAccess,pl:mixin:A}
	at net.minecraft.server.level.ServerChunkCache.m_8490_(ServerChunkCache.java:355) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.level.ServerChunkCache.m_201698_(ServerChunkCache.java:309) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:281) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:supplementaries.mixins.json:ServerLevelMixin,pl:mixin:APP:botania_xplat.mixins.json:MixinServerLevel,pl:mixin:APP:immersiveengineering.mixins.json:coremods.ServerWorldMixin,pl:mixin:APP:byg.mixins.json:common.world.MixinServerLevel,pl:mixin:APP:observable.common.json:ServerLevelMixin,pl:mixin:APP:betterdeserttemples.mixins.json:ServerLevelMixin,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:902) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:terrablender.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:326) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:842) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:terrablender.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:685) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:terrablender.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:261) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:terrablender.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at java.lang.Thread.run(Thread.java:833) [?:?] {re:mixin}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at shadows.apotheosis.adventure.boss.BossEvents.lambda$naturalBosses$0(BossEvents.java:60) ~[Apotheosis-1.18.2-5.6.1.jar%2355!/:5.6.1] {re:classloading}
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?] {re:mixin}
	at shadows.apotheosis.adventure.boss.BossEvents.naturalBosses(BossEvents.java:56) ~[Apotheosis-1.18.2-5.6.1.jar%2355!/:5.6.1] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_1834_BossEvents_naturalBosses_CheckSpawn.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.event.ForgeEventFactory.canEntitySpawn(ForgeEventFactory.java:196) ~[forge-1.18.2-40.1.80-universal.jar%23366!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraftforge.common.ForgeHooks.canEntitySpawn(ForgeHooks.java:1109) ~[forge-1.18.2-40.1.80-universal.jar%23366!/:?] {re:classloading}
	at net.minecraft.world.level.NaturalSpawner.m_47038_(NaturalSpawner.java:183) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,re:classloading,pl:mixin:APP:botania_xplat.mixins.json:MixinNaturalSpawner,pl:mixin:APP:enhancedcelestials.mixins.json:MixinWorldEntitySpawner,pl:mixin:APP:enhancedcelestials.mixins.json:access.WorldEntitySpawnerAccess,pl:mixin:A}
	at net.minecraft.world.level.NaturalSpawner.m_47045_(NaturalSpawner.java:126) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,re:classloading,pl:mixin:APP:botania_xplat.mixins.json:MixinNaturalSpawner,pl:mixin:APP:enhancedcelestials.mixins.json:MixinWorldEntitySpawner,pl:mixin:APP:enhancedcelestials.mixins.json:access.WorldEntitySpawnerAccess,pl:mixin:A}
	at net.minecraft.world.level.NaturalSpawner.m_47029_(NaturalSpawner.java:116) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,re:classloading,pl:mixin:APP:botania_xplat.mixins.json:MixinNaturalSpawner,pl:mixin:APP:enhancedcelestials.mixins.json:MixinWorldEntitySpawner,pl:mixin:APP:enhancedcelestials.mixins.json:access.WorldEntitySpawnerAccess,pl:mixin:A}
	at net.minecraft.server.level.ServerChunkCache.m_8490_(ServerChunkCache.java:355) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.level.ServerChunkCache.m_201698_(ServerChunkCache.java:309) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:281) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:supplementaries.mixins.json:ServerLevelMixin,pl:mixin:APP:botania_xplat.mixins.json:MixinServerLevel,pl:mixin:APP:immersiveengineering.mixins.json:coremods.ServerWorldMixin,pl:mixin:APP:byg.mixins.json:common.world.MixinServerLevel,pl:mixin:APP:observable.common.json:ServerLevelMixin,pl:mixin:APP:betterdeserttemples.mixins.json:ServerLevelMixin,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:A}
-- Affected level --
Details:
	All players: 1 total; [ServerPlayer['Painsoul'/170084, l='ServerLevel[world]', x=-68.63, y=26.99, z=152.69]]
	Chunk stats: 2401
	Level dimension: minecraft:the_nether
	Derived: true
	Level spawn location: World: (-48,75,928), Section: (at 0,11,0 in -3,4,58; chunk contains blocks -48,0,928 to -33,255,943), Region: (-1,1; contains chunks -32,32 to -1,63, blocks -512,0,512 to -1,255,1023)
	Level time: 31043916 game time, 31986978 day time
	Level name: world
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Level weather: Rain time: 62195 (now: false), thunder time: 36902 (now: false)
	Known server brands: forge
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
Stacktrace:
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:902) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:terrablender.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:326) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:842) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:terrablender.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:685) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:terrablender.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:261) ~[server-1.18.2-20220404.173914-srg.jar%23361!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:terrablender.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:A}
	at java.lang.Thread.run(Thread.java:833) [?:?] {re:mixin}
```

**Comment by xXjojojXx** [comment]
also have exactly the same problem

**Comment by Shadows-of-Fire** [comment]
Fixed for 5.7.0

**Comment by BugTest85** [comment]
when will 5.7.0 released ?
